content/en/guides/core/registry/configure_registry.md

```diff
@@ -39,89 +39,132 @@ A registry admin can remove individual users or entire teams from a registry. To
+| Download files from an artifact's file viewer                  | Read             |                   |   ✓    |   ✓    |   ✓   |
+| Search a registry                                              | Read             |        ✓         |   ✓    |   ✓    |   ✓   |
+| View a registry's settings and user list                       | Read             |        ✓         |   ✓    |   ✓    |   ✓   |
+| Create a new automation for a collection                       | Create           |                   |        |   ✓    |   ✓   |
+| Turn on Slack notifications for new version being added        | Create           |                   |        |   ✓    |   ✓   |
+| Create a new collection                                        | Create           |                   |        |   ✓    |   ✓   |
+| Create a new custom registry                                   | Create           |                   |        |   ✓    |   ✓   |
+| Edit collection card (description)                             | Update           |                   |        |   ✓    |   ✓   |
+| Edit linked artifact description                               | Update           |                   |        |   ✓    |   ✓   |
+| Add or delete a collection's tag                               | Update           |                   |        |   ✓    |   ✓   |
+| Add or delete an alias from a linked artifact                  | Update           |                   |        |   ✓    |   ✓   |
+| Link a new artifact                                            | Update           |                   |        |   ✓    |   ✓   |
+| Edit allowed types list for a registry                         | Update           |                   |        |   ✓    |   ✓   |
+| Edit custom registry name                                      | Update           |                   |        |   ✓    |   ✓   |
+| Delete a collection                                            | Delete           |                   |        |   ✓    |   ✓   |
+| Delete an automation                                           | Delete           |                   |        |   ✓    |   ✓   |
+| Unlink an artifact from a registry                             | Delete           |                   |        |   ✓    |   ✓   |
+| Edit accepted artifact types for a registry                    | Admin            |                   |        |        |   ✓   |
+| Change registry visibility (Organization or Restricted)        | Admin            |                   |        |        |   ✓   |
+| Add users to a registry                                        | Admin            |                   |        |        |   ✓   |
+| Assign or change a user's role in a registry                   | Admin            |                   |        |        |   ✓   |
+
+
+### Effective (inherited) Registry role
+When viewing the registry's membership list, the W&B App shows each user's effective registry role next to the role dropdown in their row.
```

Comment:
```suggestion
When viewing the registry's membership list, the W&B App shows each user's effective (inherited) registry role next to the role dropdown in their row.
```

Reply:
Your two suggestions on line 133 are a bit in conflict with each other and I can't figure out what end result you would prefer. Can you take a look?

Reply:
Merged into the next suggestion.